Python/pywarpx/picmi.py

```diff
@@ -581,6 +581,13 @@ class CylindricalGrid(picmistandard.PICMI_CylindricalGrid):
     warpx_reflect_all_velocities: bool default=False
         Whether the sign of all of the particle velocities are changed upon
         reflection on a boundary, or only the velocity normal to the surface
+
+    warpx_start_moving_window_step: int, default=0
+        The timestep at which the moving window starts
+
+    warpx_end_moving_window_step: int, default=-1
+        The timestep at which the moving window ends. If -1, the moving window
+    will continue until the end of the simulation.
```

Comment:
One last detail - can you fix the indentation?

Reply:
Thanks for this check, the indentation has been fixed.